docs/public-networks/how-to/use-configuration-file/profile.md

```diff
@@ -80,3 +86,20 @@ besu --profile=private
 :::note
 `enterprise` and `private` are aliases for the same profile.
 :::
+
+## Load external profiles
+
+You can use external profiles to create custom Besu bundles with various plugins and their default options.
+
+Add external profiles to a `profiles` directory under the root Besu directory.
+Run Besu with [`--profile`](../../reference/cli/options.md#profile) set to the external profile.
+For example, to load the `profiles/custom_profile.toml` profile, run:
+
+```bash
+besu --profile=custom_profile.toml
```

Comment:
When file name is `profiles/custom_profile.toml`, the argument to `--profile` should be `custom_profile` without the `.toml` extension.

Reply:
i.e. `besu --profile=custom_profile`